SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -71,7 +71,7 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 	AActor* Actor = Channel->Actor;
 	UClass* Class = Actor->GetClass();
 
-	FString ClientWorkerAttribute = improbable::GetOwnerWorkerAttribute(Actor);
+	FString ClientWorkerAttribute = SpatialGDK::GetOwnerWorkerAttribute(Actor);
```

Comment:
```suggestion
	FString ClientWorkerAttribute = GetOwnerWorkerAttribute(Actor);
```
This as well as other instances of `SpatialGDK` in this file seem unnecessary since we are `using namespace SpatialGDK`.

Reply:
Looks like your PR addresses this.